src/bundle/Resources/public/js/scripts/admin.content.edit.js

```diff
@@ -99,6 +99,50 @@
         }
     };
 
+    const isAutosaveEnabled = () => {
+        const autosaveSubmitBtn = form.querySelector('[name="ezplatform_content_forms_content_edit[autosave]"]');
+
+        return eZ.adminUiConfig.autosave.enabled && autosaveSubmitBtn;
```

Comment:
Could change order an little, as when `eZ.adminUiConfig.autosave.enabled` is false it's unnecessary to get data from DOM